### PR TITLE
511 extending clouseau plugin doc

### PR DESCRIPTION
**Pull request by bpluly**
Added a sentence n explicitly setting the classpath in te server parameters when running the service.
## Overview
Issue `[511](https://github.com/apache/couchdb-documentation/issues/511)` asks for elaboration of the Search Plugin documentation page. This pull request amends the current documentation.

## Testing recommendations
Check out the 511-extend-clouseau-pugin-doc branch and run Sphinx to generate the documentation. 

`make check` should have no output.

Go to the couchdb-documentation/build/html/install/search.html#install-search and see the result in the browser.

## GitHub issue number

Fixes [511](https://github.com/apache/couchdb-documentation/issues/511)

## Checklist

- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: src/install/search.rst [hunk 1]
```diff
@@ -33,7 +33,16 @@ Installation of Binary Packages
 
 Binary packages that bundle all the necessary dependencies of the search plugin are
 available on `GitHub`_.  The files in each release should be unpacked into a directory on
-the Java classpath. The service expects to find a couple of configuration files
+the Java classpath. If you do not have a classpath already set, or you wish to explicitly
+set the classpath location for Clouseau, then add the line::
+
+-classpath '/path/to/clouseau/*'
```
Comment (wohali) [inline src/install/search.rst:39]: Please indent this line by 4 spaces to ensure it is correctly formatted.

File: src/install/search.rst [hunk 2]
```diff
@@ -33,7 +33,16 @@ Installation of Binary Packages
...
+
+to the server command below. If clouseau is installed in /opt/clouseau the line would be::
+
+-classpath '/opt/clouseau'
```
Comment (wohali) [inline src/install/search.rst:43]: Please indent this line by 4 spaces to ensure it is correctly formatted.

File: src/install/search.rst [hunk 3]
```diff
@@ -33,7 +33,16 @@ Installation of Binary Packages
 
 Binary packages that bundle all the necessary dependencies of the search plugin are
 available on `GitHub`_.  The files in each release should be unpacked into a directory on
-the Java classpath. The service expects to find a couple of configuration files
+the Java classpath. If you do not have a classpath already set, or you wish to explicitly
+set the classpath location for Clouseau, then add the line::
+
+-classpath '/path/to/clouseau/*'
+
+to the server command below. If clouseau is installed in /opt/clouseau the line would be::
```
Comment (flimzy) [inline src/install/search.rst:41]:
```suggestion
to the server command below. If clouseau is installed in ``/opt/clouseau`` the line would be::
```

**Comment by bpluly** [comment]
> 
> 
> Hey @bpluly ! Thanks for the PR. Would you make this small formatting change? Then, we can get this merged.

np, should be done.

File: src/install/search.rst (hunk 1 above)
Comment (bpluly) [reply src/install/search.rst:39]: I may have misunderstood as I just accepted the suggestion. Which line should be indented?

Comment (wohali) [reply src/install/search.rst:39]: Lines 39 and 43 need to be indented by 4 spaces.

File: src/install/search.rst (hunk 2 above)
Comment (bpluly) [reply src/install/search.rst]: I think I figured out which lines.

File: src/install/search.rst (hunk 1 above)
Comment (bpluly) [reply src/install/search.rst]: All done.

**Pull request merged**